### PR TITLE
Deprecate transform(), replace with call.

### PR DESCRIPTION
**Pull request by andyferris**

### CODE REVIEW

File: REQUIRE [hunk 1]
```diff
@@ -1,3 +1,4 @@
 julia 0.4
+Compat
```
Comment (c42f) [inline REQUIRE]: Needs at least Compat version 0.8.0 for call syntax

**Comment by andyferris** [comment]
Can I merge?

**Comment by c42f** [comment]
Merge it!

File: REQUIRE (hunk 1 above)
Comment (c42f) [inline REQUIRE]: @andyferris turns out it was worth fixing this!  I just got the following warning:

```
julia> Pkg.clone("https://github.com/c42f/Displaz.jl.git")
INFO: Cloning Displaz from https://github.com/c42f/Displaz.jl.git
INFO: Computing changes...
WARNING: Compat is fixed at 0.7.20+ conflicting with requirement for CoordinateTransformations: [0.8.0,∞)
INFO: No packages to install, update or remove
```

**Pull request merged**